packages/emoji-mart/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "module": "commonjs",
+    "allowJs": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "experimentalDecorators": true,
```

Comment:
This repo doesn't use them and I wouldn't recommend enabling them without a need. TC39 is going to make a different version any .. year now.

```suggestion
```